### DISCUSSION TITLE
How do you add an icon to a template?

### DISCUSSION THREAD

**Discussion by NullVoxPopuli** [codesandbox/codesandbox-client#4667] [General] [answered]
I see a lot of templates with icons
![image](https://user-images.githubusercontent.com/199018/89732211-497b2200-da1b-11ea-8de1-632b530154b3.png)

when I make a template, how do I set the icon/picture?

**Comment by oskarabc** [comment, marked as answer]
Hello @NullVoxPopuli custom icons can't be set as of now. Icons you are seeing are determined by the original template chosen (typically dependent on underlying framework). Increased customizability is on our todo list, but do have other priorities (such as Team Pro).

**Reply by Lohoyo** [reply]
Hello! Can custom icons be set now?

**Reply by sannek** [reply]
Hi, no this is unfortunately not possible yet. They are still determined by the template/framework, and I'm not sure if/when this will actually be available.

**Reply by Lohoyo** [reply]
I got it! Thank you!
